libs/cdk/src/lib/template-management/list-manager.ts

```diff
@@ -0,0 +1,191 @@
+import {
+  EmbeddedViewRef,
+  IterableDiffer,
+  IterableDiffers,
+  NgIterable,
+  TemplateRef,
+  TrackByFunction,
+} from '@angular/core';
+import { combineLatest, Observable, OperatorFunction } from 'rxjs';
+import { filter, map, switchMap, tap, withLatestFrom } from 'rxjs/operators';
+import { StrategyCredentials } from '../render-strategies/model';
+import { onStrategy } from '../render-strategies/utils';
+
+import {
+  DistinctByFunction,
+  ListChange,
+  NgViewContext,
+  RenderSettings,
+  RxListViewContext,
+  TemplateSettings
+} from './model';
+import {
+  getChangesArray,
+  getListTemplateManager,
+  getTNode,
+  notifyAllParentsIfNeeded,
+  notifyInjectingParentIfNeeded,
+  strategyHandling,
+  TNode,
+} from './utils';
+
+export interface ListManager<T, C> {
+  nextStrategy: (config: string | Observable<string>) => void;
+
+  render(changes$: Observable<NgIterable<T>>): Observable<any>;
+}
```

Comment:
Shoul be `Observable<C>` instead of `Observable<any>`?

Reply:
fixed